wsproto/__init__.py

```diff
@@ -5,17 +5,27 @@
 
 A WebSocket implementation.
 """
+from enum import Enum
+
 from .connection import ConnectionType
-from .handshake import H11Handshake
+from .handshake import  H2Handshake, H11Handshake
```

Comment:
double space after import - or is this added by black/linting?

Reply:
No, just a mistake. Interesting that nothing fixed it...